sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -264,13 +264,15 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
    * @param locationId location id to filter.
    * @param includeInActivePY includeInActivePY, if includeInActivePY, then both active
    * and not active program year is considered.
+   * @param selectedOfferingIntensity offering intensity selected by student.
+   * @param programYearId program id to be filtered.
    * @returns program offerings for location.
    */
   async getProgramOfferingsForLocation(
     locationId: number,
     programId: number,
     programYearId: number,
-    selectedIntensity?: OfferingIntensity,
+    selectedOfferingIntensity: OfferingIntensity,
```

Comment:
Can we please rename this variable to just `selectedOfferingIntensity`? _selected_ seems to be too much attached to the UI context.

Reply:
I do not see this one changed or a reply to it. Is it going to be changed?

Reply:
I will update again.